data/abilities.js

```diff
@@ -282,7 +282,17 @@ exports.BattleAbilities = {
 		desc: "If this Pokemon eats a Berry, it restores 1/3 of its maximum HP, rounded down, in addition to the Berry's effect.",
 		shortDesc: "If this Pokemon eats a Berry, it restores 1/3 of its max HP after the Berry's effect.",
 		onEatItem: function (item, pokemon) {
-			this.heal(pokemon.maxhp / 3);
+			let targets = pokemon.side.foe.active;
+			let unnervecheck = false;
```

Comment:
`unnervecheck` is a bad variable name that doesn't conform to our standards, anyway.

I would normally suggest `foeHasUnnerve`